.shippable.yml

```diff
@@ -21,7 +21,7 @@ build:
         - ${SHIPPABLE_BUILD_DIR}/ccache
     pre_ci_boot:
         image_name: zephyrprojectrtos/ci
-        image_tag: v0.7
+        image_tag: v0.8.0-rc1
```

Comment:
Made a v0.8.0 release with west 0.6.0

Reply:
I've updated this to use the v0.8.0 tag now that I see this is up on docker hub. Ready for this to merge once CI passes?